apps/modernization-ui/src/apps/patient/add/extended/AddPatientExtendedForm.tsx

```diff
@@ -10,46 +10,42 @@ import { RaceMultiEntry } from './inputs/race/RaceMultiEntry';
 import { RaceEntry } from 'apps/patient/profile/race/RaceEntry';
 import { NameEntry } from 'apps/patient/profile/names/NameEntry';
 import { NameMultiEntry } from './inputs/Name/NameMultiEntry';
-import { AdministrativeSingleEntry } from './inputs/administrative/AdministrativeSingleEntry';
-import { AdministrativeEntry } from 'apps/patient/profile/administrative/AdministrativeEntry';
+import { Administrative } from './inputs/administrative/Administrative';
 
 type ExtendedPatientCreationForm = {
     address: AddressFields[];
     phone: PhoneEmailFields[];
     race: RaceEntry[];
     name: NameEntry[];
-    administrative: AdministrativeEntry;
+    asOf: string | null | undefined;
+    comments?: string | null;
 };
 
 type DirtyState = {
     address: boolean;
     phone: boolean;
     name: boolean;
     race: boolean;
-    administrative: boolean;
 };
 export const AddPatientExtendedForm = () => {
     const form = useForm<ExtendedPatientCreationForm>({ defaultValues: { phone: [] } });
     const [dirtyState, setDirtyState] = useState<DirtyState>({
         address: false,
-
         phone: false,
-
         race: false,
-        name: false,
-        administrative: false
+        name: false
     });
 
     return (
         <>
             <div className={styles.addPatientForm}>
                 <FormProvider {...form}>
                     <div className={styles.formContent}>
-                        <AdministrativeSingleEntry
+                        <Administrative
                             onChange={(data) => {
-                                form.setValue('administrative', data);
+                                form.setValue('asOf', data.asOf);
+                                form.setValue('comments', data.comment);
```

Comment:
issue:  There is no need to keep two forms in sync, the main form can be used here